### ISSUE TITLE
Method add of DyTasks fail with parameter in NULL

### ISSUE THREAD

**Issue by oruam85** [yegor256/thindeck#491]
The method add of DyTasks return a NullPointerException using Attributes in NULL;

Sample:

```
final Repo repo = new MkRepo();`
   	final Region region = DyTasksTest.region(repo.name());`
   	final String command = "command";`
  	DyTasks tasks = new DyTasks(region, repo);`
   	Task task = tasks.add(command,null);`
```

Remember remove the Ignore annotation od the test addTaskNullAttributes

**Comment by davvd** [comment]
I'll find a developer for the task soon...

**Comment by davvd** [comment]
@oruam85 many thanks for the report, I topped your account for 15 mins, transaction 49978795

**Comment by davvd** [comment]
@wentwog please go ahead with this task, it's yours

**Comment by wentwog** [comment]
@davvd PL #506

**Comment by wentwog** [comment]
@oruam85 the PL is merged, could you close the ticket?

**Comment by davvd** [comment]
once `491-fa4a100f` puzzle is resolved, this ticket will be complete

**Comment by wentwog** [comment]
@davvd sorry, I don't see this puzzle in the code, neither the commit with `fa4a100f` sha.
What do you mean?

**Comment by oruam85** [comment]
@wentwog this ticket can not be closed before that the bug https://github.com/yegor256/thindeck/issues/492.

**Comment by davvd** [comment]
@wentwog you're working with this issue for 15 days already... -30 added to your rating, current score is: [+125](http://www.netbout.com/b/35104?open=rating)

**Comment by wentwog** [comment]
@davvd I think you may consider to not to apply penalties in the current situation, since you had decided not to close this task when the PL was merged and created #508. So, this task is effectively waiting for #508.
And, according to §11 of the [policy](http://www.teamed.io/policy.html), you may consider to not to apply penalty if you have accepted the waiting status.
But it's up to you to decide. Have a good day )

**Comment by yegor256** [comment]
@davvd looks like we're waiting here for #492

**Comment by davvd** [comment]
> @davvd I think you may consider to not to apply penalties in the current situation, since you had decided not to close this task when the PL was merged and created #508. So, this task is effectively waiting for #508.
> And, according to §11 of the [policy](http://www.teamed.io/policy.html), you may consider to not to apply penalty if you have accepted the waiting status.
> But it's up to you to decide. Have a good day )

@wentwog ok, let's wait for #508

**Comment by wentwog** [comment]
@davvd #508 is done, please, close the ticket.

**Comment by yegor256** [comment]
@wentwog the task will be done when it's closed by @oruam85

**Comment by wentwog** [comment]
@oruam85 #508 is done, please, close the ticket.

**Comment by oruam85** [comment]
@wentwog thanks, this bug is closed now

**Comment by davvd** [comment]
@wentwog done, I added **30 mins** in payment `51273350`; +30 added to your rating, at the moment it is: [+155](http://www.netbout.com/b/35104?open=rating)

**Comment by davvd** [comment]
last puzzled solved `491-fa4a100f`/#508
